lib/os.js

```diff
@@ -60,28 +60,48 @@ const {
   setPriority: _setPriority,
 } = internalBinding('os');
 
-function getCheckedFunction(fn) {
```

Comment:
AFAICT we should be able to make the patch smaller and continue to keep things DRY by keeping this function and simply modifying its code to not use `...args`, as that seems to be the only difference?

Reply:
I think separating them bis better.

Reply:
applied